tests/test_session.py

```diff
@@ -564,6 +564,10 @@ def test_it(self):
 
 
 class TestPickleSerializer(unittest.TestCase):
+    """
```

Comment:
I don't think it is necessary to add a docstring to tests... but meh.

Reply:
I only put it there so it would pop up faster on the list of things to delete when the actual removal takes place.

Reply:
When it is removed, wouldn't the test fail?

Reply:
@jvanasco Yeah, I figured it was to make it simpler to grep across the codebase for stuff that is removed. It's fine :-)